Small fixes to automated branch creation on Issue assignment (#1190)
helmholtz-analytics/heat: quick_start.md

## New Contributors

> 1. Pick an Issue you'd like to work on. Check out [Good First Issues](https://github.com/helmholtz-analytics/heat/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22), start from most recent. Feel free to assign to yourself.

Reviewer: Can externals assign themselves? When I wanted to assign Saj to the FFT, he had to comment first in order to appear in the list of people I could assign... it seems that he could not assign himself and that I couldnt assign externals who did not already appear in the issue (e.g. by commenting etc.)

Author: ```suggestion
1. Pick an Issue you'd like to work on. Check out [Good First Issues](https://github.com/helmholtz-analytics/heat/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22), start from most recent. Get in touch and ask to be assigned to the issue.
```

Author: Good catch, addressed, thanks Fabian!
